jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/jvm/impl/JvmServiceRestImpl.java

```diff
@@ -86,10 +86,10 @@ public Response updateJvm(final JsonUpdateJvm aJvmToUpdate, final boolean update
     }
 
     @Override
-    public Response removeJvm(final Identifier<Jvm> aJvmId, final AuthenticatedUser user) {
-        LOGGER.info("Delete JVM requested: {} by user {}", aJvmId, user.getUser().getId());
-        jvmService.removeJvm(aJvmId, user.getUser());
-        return ResponseBuilder.ok();
+    public Response deleteJvm(final Identifier<Jvm> id, boolean hardDelete, final AuthenticatedUser user) {
+        LOGGER.info("Delete JVM requested: {} by user {}", id, user.getUser().getId());
+        jvmService.deleteJvm(id, hardDelete, user.getUser());
+        return Response.noContent().build();
```

Comment:
I also need to add a new test case for postman - hardDelete=true which deletes the service